src/test/openads/infrastructure/position/InMemoryPositionRepositoryTest.js

```diff
@@ -160,11 +192,15 @@ describe('InMemory Position Repository', function() {
             {
               id: '43',
               name: 'lala43',
-              source: 'google',
-              placement: 'jarjar',
-              segmentation: 'lala=true',
-              sizes: [],
-              native: {},
+              specification: {
+                source: 'google',
```

Comment:
I wonder if '**connector**' would not be a better naming for this property, instead of '**source**'. When we will include '**appnexus-prebid**' as a source, we will be refereing in fact to the connector not the AdServer. Wouldn't we?

Reply:
Not really, we won't add Prebid as a source :) 

Prebid will be added as an extra parameter of the specific connector for AppNexus as the Ad Server SDK (source). 
The specification for that connector requests should be like:

```
specification: {
  source: 'AppNexus', // the name you give to your source connector instance when you init OpenAds
  appnexus: {
    // the appnexus specific data for the request
  },
  prebid: {
    // the prebid specific data for the request
  }
}
```